samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -229,6 +267,37 @@ If you are using a different board or build target, or would like to use a custo
 
 Enable :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` to use an external antenna.
 
+.. _nrf_cloud_mqtt_multi_service_customizing_LED_status_indication:
+
+Customizing LED status indication
+=================================
+
+To disable LED status indication after a connection to nrf Cloud has been established at least once, disable :ref:`CONFIG_LED_VERBOSE_INDICATION <CONFIG_LED_VERBOSE_INDICATION>`.
+
+To make the LED turn off while the sample is idle rather than show an idle pattern, disable :ref:`CONFIG_LED_CONTINUOUS_INDICATION <CONFIG_LED_CONTINUOUS_INDICATION>`.
```

Comment:
```suggestion
To turn the LED off while the sample is idle (rather than show an idle pattern), disable :ref:`CONFIG_LED_CONTINUOUS_INDICATION <CONFIG_LED_CONTINUOUS_INDICATION>`.
```